sui/src/generate_json_rpc_spec.rs

```diff
@@ -77,6 +84,32 @@ async fn main() {
     }
 }
 
+fn replace_all_ids(s: String) -> String {
+    let s = Regex::new(r"0x[a-f\d]{40}")
```

Comment:
can we make all these regex strings into constant variables with descriptive names? just for the sake of ease of reading in the future.

i.e. `r"0x[a-f\d]{40}"` describes a 20 byte hex string, probably an address - we could name it like `ADDRESS_REGEX =  r"0x[a-f\d]{40}")`, if we don't already have this somewhere else that we can re-use.

If that sounds good to you, let's apply the same to the regexes below this as well